python/paddle/fluid/executor.py

```diff
@@ -27,6 +27,8 @@
 __all__ = ['Executor', 'global_scope', 'scope_guard']
 
 g_scope = core.Scope()
+InferNativeConfig = core.NativeConfig
+InferAnalysisConfig = core.AnalysisConfig
```

Comment:
What do you need 2 Config? Can it be done in one config?

Reply:
The `AnalysisConfig` and `NativeConfig` are exposed from C++ inference predictor directly, we hope the python API works as similar to the c++ inference as possible, so the existing inference users will be more clear facing the new python APIs.

The `AnalysisConfig` will create AnalysisPredictor while the NativeConfig will create NativePredictor, that is a quite common concept in the C++ inference.

Reply:
The two `Config` class correspond to `NativePredictor` and `AnalysisPredictor`. There is no class inheritance relations.

Reply:
Then they should be wrapped in one config?

Reply:
I don't think so.

For inference, it important is to make C++ APIs concepts consistent in python, make the python inference program consistent with the C++ version.
To wrap the existing widely used C++ program and pretend to be something else is not the first goal.

If these APIs are not suitable for the executor, we can just expose the pybind version, make a prefix with each API like `__` to warn users not to use it.

The current `PyNativeConfig` will be something like `__NativeConfig`, and we can write some testing program for convenient.